src/main/java/de/bonndan/nivio/input/Indexer.java

```diff
@@ -87,42 +88,45 @@ private void runResolvers(LandscapeDescription input, Landscape landscape) {
         KPIFactory kpiFactory = new KPIFactory();
         landscape.setKpis(kpiFactory.getConfiguredKPIs(input.getConfig().getKPIs()));
 
-        Map<ItemDescription, List<String>> templatesAndTargets = new HashMap<>();
         // read all input sources
-        new SourceReferencesResolver(formatFactory, logger).resolve(input, templatesAndTargets);
+        new SourceReferencesResolver(formatFactory, logger).resolve(input);
 
-        // apply template values to the items
-        new TemplateResolver().processTemplates(input, templatesAndTargets);
+        // apply template values to items
+        new TemplateResolver(logger).resolve(input);
 
-        // read special labels on items and assign the values to fields
-        new LabelToFieldProcessor(logger).process(input, landscape);
+        // resolve links on components to gather more data.
+        new LinksResolver(logger, linkHandlerFactory).process(input);
```

Comment:
Shouldn't the method be named `resolve()`?

Reply:
excellent observation, Sherlock!

Reply:
done.